Akvis18 assignment 2

A-Kvisgaard opened the pull request.

A-Kvisgaard committed:
Create Akvis18.txt

A-Kvisgaard committed:
Create HTML_exercise.html

A-Kvisgaard committed:
GItignore

A-Kvisgaard committed:
Merge branch 'master' into Akvis18

A-Kvisgaard committed:
Simple login and out

A-Kvisgaard committed:
Update .gitignore

A-Kvisgaard committed:
Get random number from server with ajax

A-Kvisgaard committed:
Init

A-Kvisgaard committed:
Upload Assignment 1

A-Kvisgaard committed:
Delete Footer.php

A-Kvisgaard committed:
Delete Template.html

A-Kvisgaard committed:
Add Test user on login try

A-Kvisgaard committed:
Merge branch 'Akvis18' of https://github.com/LangeHenrik/SDU-ITI-F20 into Akvis18

A-Kvisgaard committed:
Avatar upload error handling

kvinkel commented:
When running your branch locally everything seems to work well and the site is intuitive. I can create new users, log in, upload an image with header and description, see which user uploaded the picture, and see a list of all users. Default charset for the database could be changed to UTF-8 in the migration file to support non-latin characters. 
Some places the break tag should be replaced with \<br/> instead of \</br> e.g. line 47 in Register.php which also has a \<labe> tag instead of \<label>

I'm running a 3000x4000 resolution and the images go outside of the division that holds them. 
A possible fix could be changing the CSS:

.post .main-image{
    margin: auto;
    display: block;
    max-width: 1300px;
    max-height: 800px;
}

To:

 .post .main-image{
    margin: auto;
    display: block;
    max-width: 100%;
    max-height: 100%;
}

Overall it looks good! :boom:

A-Kvisgaard committed:
Review changes

A-Kvisgaard committed:
Added TestUser.txt

A-Kvisgaard committed:
Create Car.php

A-Kvisgaard committed:
Copied mvc template

A-Kvisgaard committed:
MVC Implemented and api working

A-Kvisgaard committed:
General improvements

+ Display errors now works with viewbag
+ FIxed some links
+ Preview avatar on registration
+ Errorhandling and XSS improvements to api
